packages/cli/src/commands/org/config/push.spec.ts

```diff
@@ -30,7 +37,44 @@ const mockedApplySnapshot = jest.fn();
 const mockedValidateSnapshot = jest.fn();
 const mockedPreviewSnapshot = jest.fn();
 const mockedLastReport = jest.fn();
-const mockedHasChangedResources = jest.fn();
+
+const getReport = (
```

Comment:
nit: This is very similar (identical?) to the mock function you are using in preview.spec.ts.

Suggestion: Export a common test utility function in `commands/org/config/mocks.ts` or something like that. 

Same comment for `getSuccessReport` and `getErrorReport`